tests/tracer/test_correlation_log_context.py

```diff
@@ -0,0 +1,133 @@
+import pytest
+import structlog
+
+from ddtrace import Tracer
+from ddtrace import config
+from ddtrace import tracer
+from ddtrace.opentracer.tracer import Tracer as OT_Tracer
+from tests.utils import override_global_config
+
+
+@pytest.fixture
+def global_config():
+    config.service = "test-service"
+    config.env = "test-env"
+    config.version = "test-version"
+    yield
+    config.service = config.env = config.version = None
+
+
+def tracer_injection(logger, log_method, event_dict):
+    """Inject tracer information into custom structlog log"""
+    correlation_log_context = tracer.get_log_correlation_context()
+
+    # add ids and configs to structlog event dictionary
+    event_dict["dd.trace_id"] = correlation_log_context.trace_id
+    event_dict["dd.span_id"] = correlation_log_context.span_id
+    event_dict["dd.env"] = correlation_log_context.env
+    event_dict["dd.service"] = correlation_log_context.service
+    event_dict["dd.version"] = correlation_log_context.version
```

Comment:
`event_dict.update(correlation_log_context)` could be nice to have.

that way if we add more properties to the log correlation then you'll get it automatically.

what do you think?

Reply:
Yeah I'm fine with that, I just followed what we recommend to customers on our docs: https://docs.datadoghq.com/tracing/connect_logs_and_traces/python/#no-standard-library-logging